client/src/documentLinkProvider.ts

```diff
@@ -19,6 +19,30 @@ export class BitbakeDocumentLinkProvider implements vscode.DocumentLinkProvider
     this.client = client
   }
 
+  public static async findFilesAndDirs (filePatterns: Array<{ base: string, pattern: string }>, dirPatterns: string[], maxFileResults?: number, token?: vscode.CancellationToken): Promise<{ foundFiles: vscode.Uri[], foundDirs: string[] }> {
+    const foundFiles: vscode.Uri[] = []
+    const foundDirs: string[] = []
+    try {
+      for (let i = 0; i < filePatterns.length; i++) {
```

Comment:
It looks like you are changing the algorithm from documentLinkProvider:resolveUris
I didn't expect you would need to change that?